### ISSUE TITLE
Validation set

### ISSUE THREAD

**Issue by nooshinmaghsoodi** [mims-harvard/UniTS#18]
It seems that you use test data as your validation set. Is it true? If it is the case, could you please what is the reason?

**Comment by gasvn** [comment]
We follow the common settings of other time series works, becuase not all datasets have the val set. Also, we didn't pick the best results based on the evaluation of the testing set. Instead, we train the model for a fixed number of epochs and report the results of last epoch.

**Comment by nooshinmaghsoodi** [comment]
Thank you!
